### PR TITLE
style: convert src/imlib.c to the new style

### PR DESCRIPTION
**Pull request by guijan**
This patch also crosses out the src/imlib.c entry in TODO.md
I also sorted the list of files to be converted in TODO.md, I should've done that from the start.

### CODE REVIEW

**Comment by guijan** [comment]
Ahh, I managed to build test the wrong branch. Anyway, thanks to the build failure of the original version of this PR caught by the CI I know what issue we have that makes changing styles automatically hard. I'll be fixing it when there are less PRs up.

File: src/imlib.c [hunk 1]
```diff
@@ -27,43 +27,41 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
...
-      fprintf(stderr, "%s", dispstr ? dispstr :
-              (getenv("DISPLAY") ? getenv("DISPLAY") : "NULL"));
-      fprintf(stderr, "]\n");
-      exit(EXIT_FAILURE);
-   }
+    const char *badstr;
+
+    disp = XOpenDisplay(dispstr);
+    if (!disp) {
+	    if ((badstr = dispstr) == NULL)
```
Comment (daltomi) [inline src/imlib.c:46]: There is a tab here.

**Comment by daltomi** [comment]
This commit does more than just code style, but it's ok sometimes one gets carried away :-)
We try not to do that.

**Comment by daltomi** [comment]
Hi @guijan
This PR is no longer required as styling is complete.
Anyway I tried to solve the conflicts because I was interested in what you did with `XOpenDisplay` but now that I think about it, the ideal would be to open another PR with those changes if you wish.
Cheers.

**Pull request closed without merging**